test/performance/io/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 seqan3_benchmark(format_fasta_benchmark.cpp)
 seqan3_benchmark(format_vienna_benchmark.cpp)
-seqan3_benchmark(lowlevel_stream_input_benchmark.cpp)
+seqan3_benchmark(lowlevel_stream_benchmark.cpp)
```

Comment:
I would prefer having seperate tests for input and output, because shorter benchmarks means you get to see the desired results quicker!

Input and output are separate for a lot of other things, too.

Reply:
Fine..